models/measure.js

```diff
@@ -226,41 +226,105 @@ module.exports = (event, state) => {
   }
 }
 
-const fetchMeasures = (params, user) => (dispatch) => {
-  const terms = params.terms && params.terms.replace(/[^\w\d ]/g, '').replace(/(hr|s) (\d+)/i, '$1$2').replace(/(\S)\s+(\S)/g, '$1 & $2')
-  const fts = terms ? `&tsv=fts(simple).${encodeURIComponent(terms)}` : ''
+const fetchMeasures = (params, cookies, geoip, query, user, location) => (dispatch) => {
 
-  const orders = {
-    upcoming: '&status=not.eq.Introduced&introduced_at=not.is.null&failed_lower_at=is.null&failed_upper_at=is.null&or=(passed_lower_at.is.null,passed_upper_at.is.null,and(passed_lower_at.is.null,passed_upper_at.is.null))&order=next_agenda_action_at.asc.nullslast,next_agenda_begins_at.asc.nullslast,next_agenda_category.asc.nullslast,last_action_at.desc.nullslast,number.desc',
-    new: '&introduced_at=not.is.null&order=introduced_at.desc,number.desc',
-    proposed: '&introduced_at=is.null&order=created_at.desc,title.asc',
-  }
+  const order = '&order=next_agenda_action_at.asc.nullslast,next_agenda_begins_at.asc.nullslast,next_agenda_category.asc.nullslast,last_action_at.desc.nullslast,created_at.desc.nullslast,number.desc'
```

Comment:
last_action_at.nullslast,created_at.nullslast

when I filter for imported and liquid bills at the same time, this code puts all Liquid bills at the bottom. How do I tell it to sort by last_action_at or created_at if last_action_at is null?